src/libproc_macro/lib.rs

```diff
@@ -69,7 +69,7 @@ pub struct LexError {
 #[stable(feature = "proc_macro_lexerror_impls", since = "1.43.0")]
 impl fmt::Display for LexError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.write_str("lex error")
+        f.write_str("cannot parse string into token stream")
```

Comment:
@petrochenkov Your suggestion felt a little wordy so I simplified it a bit. What do you think of this?

Reply:
It's ok.